resources/docs/apps/app-details.md

```diff
@@ -1,5 +0,0 @@
-This service is used by the DE to obtain high-level details about a single App.
-
-#### Delegates to metadata service
-    POST /ontologies/{ontology-version}/filter
-Please see the metadata service documentation for information about the `hierarchies` response field.
```

Comment:
By importing the `OntologyHierarchy` and `OntologyHierarchyList` schemas from the metadata service for use in both the apps and metadata services, we no longer need docs that say "Please see the metadata service documentation for response information." 🎉